src/GHC/Util.hs

```diff
@@ -15,8 +17,13 @@ module GHC.Util (
 
   , parsePragmasIntoDynFlags
   , parseFileGhcLib, parseExpGhcLib, parseImportGhcLib
+  , pattern SrcSpan, srcSpanFilename, srcSpanStartLin, srcSpanStartColumn, srcSpanEndLin, srcSpanEndColumn
+  , pattern SrcLoc, srcFilename, srcLine, srcColumn
+  , showSrcLoc'
   ) where
 
+import Data.List.Extra (drop1)
```

Comment:
No need to import this with restricted functions.

Reply:
I tend to prefer it by default because it's way easier for people without an ide to navigate code. (from my past experience)

Reply:
Easier solution is to get an IDE: https://github.com/digital-asset/ghcide

Reply:
Yes absolutely! `ghcide` has been invaluable for working on `hlint`, comparing my experience to the one I had previously without relying on `ghcide`! :heart: